mysql_mimic/session.py

```diff
@@ -164,13 +187,13 @@ class Session(BaseSession):
     dialect: Type[Dialect] = MySQL
 
     def __init__(self, variables: Variables | None = None):
+        self._variable_processor: VariableProcessor
```

Comment:
I had missed the timestamp dependency. Lets do it like you had before, just instantiate it when we need it.